### PR TITLE
Review audit certificate

### PR DESCRIPTION
**Pull request by jaleszek**
Reviewing the certificate functionality.

Re: `+    t.integer  "form_answer_id",      null: false`

I see:

```
/home/leszek/Desktop/www/qae/db/migrate/20150228145247_create_audit_certificates.rb:
    1  class CreateAuditCertificates < ActiveRecord::Migration
    2    def change
    3:     create_table :audit_certificates do |t|
    4        t.references :form_answer, null: false, index: true
```

so seems that change above is fine.

### CODE REVIEW

File: app/assets/javascripts/admin/form_answers.js.coffee [hunk 1]
```diff
@@ -6,6 +6,17 @@ ready = ->
...
   bindRags("#section-case-summary .edit_assessor_assignment")
 
+  $("#new_review_audit_certificate").on "ajax:success", (e, data, status, xhr) ->
+    $(this).find(".form-group").removeClass("form-edit")
+    $(this).find(".form-edit-link").remove()
+    area = $(".audit-cert-description textarea")
+    unless area.val()
+      $(this).find(".form-value").html($("<p>No change necessary</p>"))
+    else
+      $(this).find(".form-value").html($("<div><label>Changes made</label><p class='control-label'>#{area.val()}</p></div>"))
```
Comment (matthewford) [inline app/assets/javascripts/admin/form_answers.js.coffee]: Line exceeds maximum allowed length

File: app/models/audit_certificate.rb [hunk 2]
```diff
@@ -3,6 +3,7 @@ class AuditCertificate < ActiveRecord::Base
 
   begin :associations
     belongs_to :form_answer
+    belongs_to :reviewable, polymorphic: true
```
Comment (matthewford) [inline app/models/audit_certificate.rb]: Inconsistent indentation detected.

File: app/models/audit_certificate.rb [hunk 3]
```diff
@@ -12,5 +13,19 @@ class AuditCertificate < ActiveRecord::Base
                            file_size: {
                              maximum: 25.megabytes.to_i
                            }
+    validates :reviewable_type,
```
Comment (matthewford) [inline app/models/audit_certificate.rb]: Inconsistent indentation detected.

File: config/routes.rb [hunk 4]
```diff
@@ -100,6 +100,7 @@
       resources :comments
       resources :form_answer_attachments, only: [:create, :show, :destroy]
       resources :feedbacks, only: [:create, :update] do
+      resources :review_audit_certificates, only: [:create]
```
Comment (matthewford) [inline config/routes.rb]: Use 2 (not 0) spaces for indentation.

**Comment by RuslanHamidullin** [comment]
Good to merge :+1:

**Pull request merged**